<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVolumesModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerVolumesModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volumes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/Volumes.png</normaloff>:/Icons/Volumes.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="ActiveVolumeLabel">
     <property name="text">
      <string>Active Volume</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="ActiveVolumeNodeSelector">
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLVolumeNode</string>
      </stringlist>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="DisplayCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="qSlicerVolumeDisplayWidget" name="VolumeDisplayWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="InfoCollapsibleButton">
     <property name="text">
      <string>Volume Information</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="qMRMLVolumeInfoWidget" name="MRMLVolumeInfoWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLVolumeInfoWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLVolumeInfoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerVolumeDisplayWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerVolumeDisplayWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerVolumesModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerVolumesModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLVolumeInfoWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumesModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ActiveVolumeNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>1</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ActiveVolumeNodeSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>VolumeDisplayWidget</receiver>
   <slot>setMRMLVolumeNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ActiveVolumeNodeSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>MRMLVolumeInfoWidget</receiver>
   <slot>setVolumeNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
